webcompat/templates/home-page/form.html

```diff
@@ -88,48 +83,47 @@ <h1 class="headline-1">Report Site Issue</h1>
         {{ subradio.label(class_='form-label label-radio') }}
       {% endfor %}
     </div>
-    <div class="form-upload form-element js-image-upload js-Form-group">
-      <label class="wc-UploadForm wc-UploadForm--homePage js-image-upload-label" for="image">
-        {{ form.image(class_='form-input input-upload wc-ButtonUpload', accept='.jpe,.jpg,.jpeg,.gif,.png,.bmp') }}
-        {% if form.image.errors %}
-          {% for error in form.image.errors %}
-          <span class="wc-Form-helpMessage--imageUpload">{{ error }}</span>
-          {% endfor %}
-        {% endif %}
-        <div class="wc-UploadForm-wrapper">
-          <label class="form-label label-upload wc-UploadForm-label">
-            <svg class="icon icon-big wc-UploadForm-icon" role="presentation">
-              <use class="icon-upload" href="#svg-upload-2">Upload</use>
-            </svg>
-            Upload screenshot
-          </label>
+    <div class="form-upload js-image-upload js-Form-group">
+        {{ form.image(class_='form-input input-upload', accept='.jpe,.jpg,.jpeg,.gif,.png,.bmp', id='image') }}
+        <div class="label-icon label-upload js-label-upload">
+          <svg class="icon icon-big" role="presentation" title="Upload Screenshot">
+            <use class="icon-upload" href="#svg-plus">Upload Screenshot</use>
+          </svg>
+          <small class="label-icon-message">Upload screenshot</small>
+        </div>
+
+        <div class="label-icon label-remove is-hidden js-remove-upload">
+          <svg class="icon icon-big" role="presentation" title="Remove Screenshot">
+            <use class="icon-remove" href="#svg-x">Remove Screenshot</use>
+          </svg>
+          <small class="label-icon-message">Remove screenshot</small>
+        </div>
+
+        
+        <div class="label-icon label-error is-hidden js-error-upload">
+          <svg class="icon icon-big" role="presentation" title="An error occoured">
+            <use class="icon-error" href="#svg-warning">An error occoured</use>
+          </svg>
+          {% if form.image.errors %}
+            {% for error in form.image.errors %}
+              <small class="label-icon-message">{{ error }}</small>
+            {% endfor %}
+          {% endif %}
         </div>
-        <button
-          class="wc-UploadForm-button is-hidden r-ResetButton"
-          role="button"
-          type="button"
-          tabindex="-1"
-        >
-          <img
-            class="wc-Upload-Loader js-Upload-Loader"
-            src="/img/upload-loader.svg"
-            alt="Uploading image"
-          >
-          Remove image upload
-        </button>
-      </label>
     </div>
+
     <div class="form-button form-element js-Form-group">
       <button
-        class="button button-primary wc-Button wc-Button--default js-Button"
+        class="button button-third js-Button"
```

Comment:
What does `button-third` mean? I don't see it anywhere else.

Reply:
Thanks for the hint. This needs to go.